modules/nw-metallb-configure-address-pool-vlan.adoc

```diff
@@ -0,0 +1,66 @@
+:_mod-docs-content-type: PROCEDURE
+[id="nw-metallb-configure-address-pool-vlan_{context}"]
+= Configure MetalLB address pool for VLAN
+
+As a cluster administrator, you can add address pools to your cluster to control the IP addresses on a created VLAN that MetalLB can assign to load-balancer services
+
+.Prerequisites
+
+* Install the OpenShift CLI (oc).
+
+* Separate VLAN configured.
+
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+. Create a file, such as `ipaddresspool-vlan.yaml`, with content such as the following example:
++
+[source,yaml]
+----
+apiVersion: metallb.io/v1beta1
+kind: IPAddressPool
+metadata:
+  namespace: metallb-system
+  name: doc-example-vlan
+  labels: 
+    zone: east <1>
+spec:
+  addresses:
+  - 192.168.100.1-192.168.100.254 <2>
+----
+<1> This label assigned to the `IPAddressPool` can be referenced by the `ipAddressPoolSelectors` in the `BGPAdvertisement` CRD to associate the `IPAddressPool` with the advertisement.
+<2> This IP range must match the subnet assigned to the VLAN on your network. To support layer 2 (L2) mode, the IP address range must be within the same subnet as the cluster nodes.
+
+. Apply the configuration for the IP address pool:
++
+[source,terminal]
+----
+$ oc apply -f ipaddresspool-vlan.yaml
+----
+
+. To ensure this configuration applies to the VLAN you need to set the `spec` `gatewayConfig.ipForwarding` to `Global`. 
+
+.. Run the following command to edit the network configuration custom resource (CR):
++
+[source,terminal]
+----
+$ oc edit network.config.openshift/cluster
+----
+
+.. Within the editor, update the `spec.defaultNetwork.ovnKubernetesConfig` section to include the `gatewayConfig.ipForwarding` set to `Global`. It should look something like this:
++
```

Comment:
Maybe add a example header above the example to state that it's an example copnfiguration?